src/test/java/myfirstproject/service/impl/PrepareDataImplTest.java

```diff
@@ -0,0 +1,33 @@
+package myfirstproject.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import myfirstproject.model.Fruit;
+import myfirstproject.service.PreparingData;
+import org.junit.Test;
+
+public class PrepareDataImplTest {
+    @Test
+    public void isEmptyData_Ok() {
+        PreparingData preparingData = new PrepareDataImpl();
+        String expected = preparingData.prepare(Collections.emptyMap());
+        String actual = "fruit,quantity" + System.lineSeparator();
```

Comment:
expected - value that should be
actual - value that method returned 
let's change names